test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -1037,6 +1040,58 @@ public void testGetTabletInformation() throws Exception {
         });
       }
 
+      var unboundedStartRange = RowRange.atMost(new Text("4"));
```

Comment:
There are 8 cases of inclusive/exclusive inf/not-inf combos and not sure they are all being covered.  Seems like a subset of those are covered.  The following is where I tried to work out what cases were covered by the current test, not sure if its correct though.  Including in case its useful.

```java
RowRange.range(null, true, Y, false);
RowRange.range(null, true, Y, true); // covered
RowRange.range(X, false, null, true);
RowRange.range(X, true, null, true); //covered
RowRange.range(X, false, Y, false);
RowRange.range(X, false, Y, true); // covered
RowRange.range(X, true, Y, false);
RowRange.range(X, true, Y, true); //covered
```